kidviz/views.py

```diff
@@ -286,6 +291,39 @@ def form_valid(self, form):
 
         return super().form_valid(form)
 
+    def _constructs(self, observation, available_constructs):
```

Comment:
Again, It looks like method from observation class.

Generally, models should be fat - take a look at makimo django style guide for reference :)